### ISSUE TITLE
Suggestions

### ISSUE THREAD

**Issue by D4NNY1000000** [Lululuz/FutBud#4]
Not sure if this is the best place, didnt see a chat option or that. 
Could keep this open for suggestions?

Buy price in logs or statistics added such as profit made on a player...

**Comment by xtheokles** [comment]
When someone bids faster than you it usually throws an Exception, perhaps it can be a good idea if it doesn't stops the bot everytime, it's just a suggestion :)

**Comment by Lululuz** [comment]
Good idea, I'll keep this thread open.

Do you know which Exception is thrown? Or is it the general one that is catched in the end?

**Comment by xtheokles** [comment]
I don't know exactly which is the exception, but stopping the bot only make sense in case you get an "Expired Session" or "Captcha triggered exception". Don't you think?

When you buy a player, you can also add the BuyNow price on the log area, just by adding "auctionInfo.BuyNow".

I've seen that you are updating the bot constantly. Good job!:D

PD: I would like to help, but for the moment I'm too busy :(

**Comment by D4NNY1000000** [comment]
Reset counters on load?
Option to play sound when player bought?

**Comment by Lululuz** [comment]
I'll see if I find time for it. I'm pretty busy atm :(

**Comment by Lululuz** [comment]
New version contains:
Option reset counters on load
Option to play sound when player bough/sold
Session profit

Bot only stops on the following Exceptions now:
CaptchaTriggeredException, FormatException, NullReferenceException

Please let me know if everything works fine for you.

**Comment by M-Denz** [comment]
Hello Lululuz,

Wanted to say great work on the project!

Anyway, what I wanted to say is that I thing that the debug feature should just be part of the main log itself TBH; without it it is hard to tell what is going on and with the debug makes it look more interesting.

**Comment by Lululuz** [comment]
Debug is now a savable setting in the new commit. Everyone who wants extra info can enable and save it so debug is enabled by default.

**Comment by xtheokles** [comment]
Good job @Lululuz , I'll notify you if I find bugs or some suggestions :)

**Comment by xtheokles** [comment]
@Lululuz what a nice job! In love with auto price! In the future it can be a good idea add search filter by Chemistry, Position and in Log when you buy a player the buy price can be displayed next to the "Buyout for xxxxxx".

**Comment by Xtremethegamer** [comment]
You could introduce also that prices refresh automatically every 30 mins or 1 hour while you're using autoprice, so that if you use it for a long period you have no danger of losing coins

**Comment by xtheokles** [comment]
@Xtremethegamer If i'm not wrong it actually does it every 5 minutes more or less

**Comment by Xtremethegamerz** [comment]
@xtheokles Does it? I never noticed so, but now that I am seeing you're right... Nevermind then, thanks for telling me ^^

**Comment by Lululuz** [comment]
Yes it does every 5 minutes. I'll maybe make it adjustable some time...
If you got some suggestions you can also put them in here so I can keep them in one place :)
https://docs.google.com/document/d/1pEpd6ZWHh5MoT6gLzjZ8u16KJ-ipcnFqWz9Q8bsNBQM/

**Comment by Xtremethegamerz** [comment]
@Lululuz Just added one ^^

**Comment by kostyan5** [comment]
I wrote my own autobuyer in Swift for OSX.  If I price search 1 per second, I get temp ban (error message) after exactly 1000 tries.

There seems to be a hard limit on short term (perhaps per hour) and long term (per day).  It would be nice if we could figure out what it is.

**Comment by Xtremethegamer** [comment]
Why don't you add groups? For example, I want two different prices for 2 groups of players, like:

Gold players: Buy 75% Sell 95%
Silver players: Buy 70% Sell 94%

Or even like 2 groups of Silver players etc

**Comment by code9monkey** [comment]
The screenshots on the README page need updated (dead links).
